mreg/api/permissions.py

```diff
@@ -1,7 +1,6 @@
 from django.conf import settings
-
 from rest_framework import exceptions
-from rest_framework.permissions import IsAuthenticated, SAFE_METHODS
+from rest_framework.permissions import SAFE_METHODS, IsAuthenticated
```

Comment:
Why changed sorting?

Reply:
Fixed in https://github.com/unioslo/mreg/pull/484/commits/3fb4213c07bbd979d25a4cf18a83dae61abdb22b